### PR TITLE
[Endgame] Prep v1.11 branch for release

### PR DESCRIPTION
**Pull request by hhunter-ms**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

- [x] Update README.md
- [x] Update version dropdown in nav bar
- [x] Supported versions section
- [x] All output logs change 1.10 to 1.11 (change dapr-latest-version.html)
- [x] Update URL in config.toml
- [x] Update workflow for docs.dapr.io to v1.11 branch

## Issue reference

PR will close: #3448

### CODE REVIEW

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 1]
```diff
@@ -45,8 +45,9 @@ The table below shows the versions of Dapr releases that have been tested togeth
 
 | Release date | Runtime     | CLI  | SDKs  | Dashboard  | Status |
 |--------------------|:--------:|:--------|---------|---------|---------|
-| May 15th 2023 | 1.10.7</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 2.5.0 | 0.11.0 | Supported (current) |
-| May 12th 2023 | 1.10.6</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 2.5.0 | 0.11.0 | Supported (current) |
+| June 7th 2023 | 1.11.0</br>  | 1.11.0 | Java 1.8.0 </br>Go 1.7.0 </br>PHP 1.1.0 </br>Python 1.10.0 </br>.NET 1.11.0 </br>JS 3.0.0 | 0.11.0 | Supported (current) |
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-release-policy.md]: Let's make this the 12th June
```suggestion
| June 12th 2023 | 1.11.0</br>  | 1.11.0 | Java 1.8.0 </br>Go 1.7.0 </br>PHP 1.1.0 </br>Python 1.10.0 </br>.NET 1.11.0 </br>JS 3.0.0 | 0.11.0 | Supported (current) |
```

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 2]
```diff
@@ -45,8 +45,9 @@ The table below shows the versions of Dapr releases that have been tested togeth
 
 | Release date | Runtime     | CLI  | SDKs  | Dashboard  | Status |
 |--------------------|:--------:|:--------|---------|---------|---------|
-| May 15th 2023 | 1.10.7</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 2.5.0 | 0.11.0 | Supported (current) |
-| May 12th 2023 | 1.10.6</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 2.5.0 | 0.11.0 | Supported (current) |
+| June 12th 2023 | 1.11.0</br>  | 1.11.0 | Java 1.8.0 </br>Go 1.7.0 </br>PHP 1.1.0 </br>Python 1.10.0 </br>.NET 1.11.0 </br>JS 3.0.0 | 0.11.0 | Supported (current) |
```
Comment (shubham1172) [inline daprdocs/content/en/operations/support/support-release-policy.md]:
```suggestion
| June 12th 2023 | 1.11.0</br>  | 1.11.0 | Java 1.8.0 </br>Go 1.7.0 </br>PHP 1.1.0 </br>Python 1.10.0 </br>.NET 1.11.0 </br>JS 3.1.0 | 0.11.0 | Supported (current) |
```

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 3]
```diff
@@ -45,8 +45,9 @@ The table below shows the versions of Dapr releases that have been tested togeth
...
+| May 15th 2023 | 1.10.7</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 2.5.0 | 0.11.0 | Supported |
+| May 12th 2023 | 1.10.6</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 2.5.0 | 0.11.0 | Supported |
 | April 13 2023 |1.10.5</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 2.5.0 | 0.11.0 | Supported (current) |
```
Comment (shubham1172) [inline daprdocs/content/en/operations/support/support-release-policy.md]:
```suggestion
| May 15th 2023 | 1.10.7</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 3.0.0 | 0.11.0 | Supported |
| May 12th 2023 | 1.10.6</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 3.0.0 | 0.11.0 | Supported |
| April 13 2023 |1.10.5</br>  | 1.10.0 | Java 1.8.0 </br>Go 1.6.0 </br>PHP 1.1.0 </br>Python 1.9.0 </br>.NET 1.10.0 </br>JS 3.0.0 | 0.11.0 | Supported (current) |
```

File: daprdocs/content/en/operations/support/support-release-policy.md (hunk 2 above)
Comment (artursouza) [reply daprdocs/content/en/operations/support/support-release-policy.md]: Java is 1.9

File: daprdocs/content/en/operations/support/support-release-policy.md (hunk 3 above)
Comment (artursouza) [reply daprdocs/content/en/operations/support/support-release-policy.md]: Java is 1.9

Comment (mukundansundar) [reply daprdocs/content/en/operations/support/support-release-policy.md]: dashboard is 0.13.0 ...

Comment (mukundansundar) [reply daprdocs/content/en/operations/support/support-release-policy.md]: go-sdk should be 1.8 for this release .... 
cc @dapr/maintainers-go-sdk

Comment (daixiang0) [reply daprdocs/content/en/operations/support/support-release-policy.md]: > go-sdk should be 1.8 for this release .... cc @dapr/maintainers-go-sdk

Created.

**Review by yaron2 — APPROVED** [review]

**Pull request merged**